src/test/regress/expected/local_table_join.out

```diff
@@ -1438,7 +1442,7 @@ set citus.local_table_join_policy to 'prefer-distributed';
 SELECT COUNT(*) FROM distributed_table d1 JOIN postgres_table using(key)
 WHERE d1.key IN (SELECT key FROM distributed_table WHERE d1.key = key and key = 5);
 DEBUG:  Wrapping relation "distributed_table" "d1" to a subquery
-DEBUG:  generating subplan XXX_1 for subquery SELECT key FROM local_table_join.distributed_table d1 WHERE true
+DEBUG:  generating subplan XXX_1 for subquery SELECT key FROM local_table_join.distributed_table d1 WHERE (key OPERATOR(pg_catalog.=) 5)
```

Comment:
We have just two lines with the same change (key OPERATOR(pg_catalog.=) 5)
How about a normalized line, we can change the table names such that the normalization rule detects the line easily.
e.g.
`postgres_table_key5_filtering`
`distributed_table_key5_filtering`

Just an idea, could be better. But I am reluctant of merging a new file with 1659 new lines just for 2 lines change in the test output. What do you think?

Reply:
Applied the solution

Reply:
Thanks, these changes make sense. Based on your solution, I think we can leverage renaming the table even further by using a table alias, this will make changes even more minimal. What I mean by this:
In order to include everything with one normalize line, and avoid creating a new table, we can make use of aliases. For example, `table_diff_filtering` alias. We can include this alias in `local_dist_join_mixed` line difference as well. And we can change the output to WHERE true
(in local_table_join, `WHERE true` is in PG16; in local_dist_join_mixed, `WHERE true` is in PG17, but we don't need to document this part)

I did a sketch commit and it seems to work. https://github.com/citusdata/citus/tree/naisila/try_filtering_diff
Could you please update this PR accordingly?

(PS: We had used this [table alias trick in PG16 support](https://github.com/citusdata/citus/pull/7112/commits/4dc7a23b2a42246490aa4f06887d40fc47cbe8ff) as well, I believe it deserves to be added to the documentation.)

Reply:
decided on another approach